pages/common/nping.md

```diff
@@ -0,0 +1,36 @@
+# nping
+
+> Network packet generation tool / ping utility.
+> More information: <https://nmap.org/nping/>.
```

Comment:
```suggestion
> Network packet generation tool/ping utility.
> More information: <https://nmap.org/nping>.
```

Reply:
I will change. But that's what it said in the man page.
```
NAME
       nping - Network packet generation tool / ping utility
```

Reply:
> But that's what it said in the man page.
> 
> ```
> NAME
>        nping - Network packet generation tool / ping utility
> ```

That is fine there because the man page is formatted in Groff whereas we follow Common Mark (Markdown) in tldr so I I don't think the space is necessary (as we don't do the same in other English pages).

Reply:
So there are several English writing styles for computer documentation? I never stop learning.